client/components/emojify/index.jsx

```diff
@@ -34,8 +38,14 @@ export default class Emojify extends PureComponent {
 	}
 
 	render() {
+		if ( 'string' === typeof this.props.children ) {
```

Comment:
I'm sure React is going to surprise us many times with the type of children. I wouldn't relay on that. On the other hand inlined `<div />` should do the trick, too :)